go.mod

```diff
@@ -5,7 +5,7 @@ go 1.13
 require (
 	github.com/aquasecurity/fanal v0.0.0-20200317181056-f28b6d21845c
 	github.com/aquasecurity/go-dep-parser v0.0.0-20190819075924-ea223f0ef24b
-	github.com/aquasecurity/trivy-db v0.0.0-20200318223623-7d3e67b057d4
+	github.com/aquasecurity/trivy-db v0.0.0-20200327012036-a11aca42bc75
```

Comment:
Note to self: Change this to master once https://github.com/aquasecurity/trivy-db/pull/32 is merged.